DragenGE.sh

```diff
@@ -209,6 +235,9 @@ if [ $expGVCF == $obsGVCF ]; then
 
     fi
 
+    # Remove lock file from dragen
+    rm /mnt/Data-MSA/raw/dragen_markers/${seqId}_*_locked
```

Comment:
Wonder if we could use the TRAP function in bash to delete the marker file if the script crashes? Not sure